src/libraries/System.Diagnostics.Process/src/Microsoft/Win32/SafeHandles/SafeProcessHandle.cs

```diff
@@ -15,6 +15,7 @@
 
 namespace Microsoft.Win32.SafeHandles
 {
+    /// <summary>Provides a managed wrapper for a process handle.</summary>
     public sealed partial class SafeProcessHandle : SafeHandleZeroOrMinusOneIsInvalid
     {
         internal static readonly SafeProcessHandle InvalidHandle = new SafeProcessHandle();
```

Comment:
Line 22: The public parameterless constructor does not show up in MS Docs, so there was no documentation to backport.
@joelmartinez do you know why this constructor can't be seen in dotnet-api-docs?
https://github.com/dotnet/dotnet-api-docs/blob/dc599c5db9e3464765b4e8c319bc22e525f4eebf/xml/Microsoft.Win32.SafeHandles/SafeProcessHandle.xml

Reply:
This is a bug. I opened an issue for the Docs dev team to address this: https://dev.azure.com/ceapex/Engineering/_workitems/edit/374279  cc @TianqiZhang